WebAPI/Controllers/ParcelController.cs

```diff
@@ -34,5 +36,12 @@ public ActionResult<Parcel> Post(Parcel parcel)
             _logger.LogInformation("Received Post request at parcel\n"+parcel.ToString()); //testing purposes
             return Ok(parcel);
         }
+        
+        [HttpGet("parcel/{id}")]
```

Comment:
Try grouping methods a bit so structure of file would be like:
Get; Get; Post. Instead of current Get Post Get